packages/webpack-extensive-lodash-replacement-plugin/index.js

```diff
@@ -332,74 +330,116 @@ const LODASH_MODULE_NAMES = [
 	'zipWith',
 ];
 
-function throwError( message, error = null ) {
-	throw new Error(
-		`[ExtensiveLodashReplacementPlugin] ${ message }${ error ? ' Error: ' : '' }${ error }`
+function createError( message, error = null ) {
+	return new Error(
+		`[ExtensiveLodashReplacementPlugin] ${ message }${ error ? ` Error: ${ error }` : '' }`
 	);
 }
 
+function getModuleForPath( moduleResolver, rootPath, packageName ) {
+	return new Promise( ( resolve, reject ) => {
+		moduleResolver.resolve(
+			{},
+			rootPath,
+			packageName,
+			{},
+			( error, resource, resourceResolveData ) => {
+				if ( error ) {
+					reject(
+						createError(
+							`Could not find module ${ packageName } for import on ${ rootPath }.`,
+							error
+						)
+					);
+				}
+
+				resolve( resourceResolveData );
+			}
+		);
+	} );
+}
+
 class ExtensiveLodashReplacementPlugin {
-	constructor( baseFile = './package.json' ) {
-		this.baseFile = path.resolve( baseFile );
-		let data;
+	constructor( baseDir = '.' ) {
```

Comment:
A better way how to configure the plugin, and more in line how other plugins are configured, is:
```js
new ExtensiveLodashReplacementPlugin( { baseDir } )
```

Reply:
Sounds good!